### PR TITLE
Bugfix/electron popup translation

### PR DESCRIPTION
**Pull request by matheustavaresdev**
### Intent

Fix an issue that was causing the Choose R Popup on Windows not to display any text. Now the language change is being properly handled on it too.

### Approach

Added the language handling mechanism for the Choose R popup as well as handling some issues that were present on this specific case scenario.

### Automated Tests

None

### QA Notes

Info on #11079

### Checklist

- [X] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [X] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [X] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [X] This PR passes all local unit tests

Addresses #11079

### CODE REVIEW

File: src/node/desktop/src/ui/utils.ts [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
-      if (localeLastTimeData.isSet) {
-        const localeData = isLocalStorageItemSet(localeStorageItemKey);
-
-        if (localeData.isSet && now > parseInt('' + localeLastTimeData.value, 10)) {
-          clearInterval(isThereNewLanguageInterval);
-
-          const newLanguage = localeData.value;
-
-          resolve('' + newLanguage);
+      // After 5 seconds, the default language will be set
```
Comment (kevinushey) [inline src/node/desktop/src/ui/utils.ts:20]: Isn't this more like "give up after 5 seconds"?

Comment (matheustavaresdev) [reply src/node/desktop/src/ui/utils.ts:20]: Yup!

File: src/node/desktop/src/main/utils.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export function getAppPath(): string {
 }
 
 export function initializeSharedSecret(): void {
-  const sharedSecret = randomUUID();
+  const sharedSecret = randomString() + randomString() + randomString();
```
[the remark this answers is not in the document]
Comment (matheustavaresdev) [reply src/node/desktop/src/main/utils.ts:44]: No, I have no idea why did this happen 😓

**Review by kevinushey — APPROVED** [review]
LGTM!

**Pull request merged**